tests/layers/test_attn_varlen_pack_layout.py

```
@@ -0,0 +1,39 @@
+import pytest
+import torch
+
+from fla.layers.attn import Attention
+from fla.utils import device
+
+
+try:
+    HAS_FLASH = True
+except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)